tests/cases/compiler/lateBoundClassMemberAssignmentJS.ts

```diff
@@ -0,0 +1,17 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @strict: true
+// @target: es6
+// @filename: file.js
+const _sym = Symbol("_sym");
+class MyClass {
+    constructor() {
+        this[_sym] = "ok";

```

Comment:
I would like two other tests just to test basic element assignment declarations:

1. one that uses properties with dashes in their names.
2. one that uses all the different kinds of assignment declarations to show off which work with element access and which don't.